ci/test_narwhals.sh

```diff
@@ -39,6 +39,26 @@ NARWHALS_POLARS_GPU=1 python -m pytest \
     --dist=worksteal \
     --constructors=polars[lazy]
 
+rapids-logger "Run narwhals tests for cuDF Pandas"
+NARWHALS_DEFAULT_CONSTRUCTORS=pandas python -m pytest \
+    -p cudf.pandas \
+    --cache-clear \
+    --junitxml="${RAPIDS_TESTS_DIR}/junit-cudf-pandas-narwhals.xml" \
+    -k "not ( \
```

Comment:
I would use another list of expected failures (as a pytest plugin) like we do when running narwhals tests with cudf, but we already pass `-p cudf.pandas`. So I'm skipping them instead.